app/services/forest_liana/users_getter.rb

```diff
@@ -0,0 +1,39 @@
+module ForestLiana
+  class UsersGetter
+    def initialize(endpoint, rendering_id)
+      @uri = URI.parse("#{forest_url}/forest/renderings/#{rendering_id}/#{endpoint}")
+    end
+
+    def perform
+      http = Net::HTTP.new(@uri.host, @uri.port)
+      http.use_ssl = true if forest_url.start_with?('https')
+
+      begin
+        http.start do |client|
+          request = Net::HTTP::Get.new(@uri.path)
+          request['Content-Type'] = 'application/json'
+          request['forest-secret-key'] = ForestLiana.env_secret
+          request['forest-jwt'] = @forest_token if @forest_token
```

Comment:
sorry to insist but If you rename it, it should be everywhere, `forest-jwt` remains here.

Reply:
yep, just saw it